File: MATEMÁTICA/2596/main.c
#include <math.h>
#include <stdio.h>

int minSpheres(int n) {
    return n - (int)sqrt(n);
}

int main() {
    int c, n;
    scanf("%d", &c);
    while (c--) {
        scanf("%d", &n);
        printf("%d\n", minSpheres(n));
    }
    return 0;
}

File: MATEMÁTICA/2596/Main.java
import java.util.Scanner;

public class Main {

    static int minSpheres(int n) {
        return n - (int) Math.sqrt(n);
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int c = scanner.nextInt();
        while (c-- > 0) {
            int n = scanner.nextInt();
            System.out.println(minSpheres(n));
        }
        scanner.close();
    }
}